src/remote/workspace.py

```diff
@@ -235,3 +264,34 @@ def clear_remote(self) -> None:
     def create_remote(self) -> None:
         """Remove remote directory"""
         self.execute(f"mkdir -p {self.remote.directory}", simple=True)
+
+    @contextmanager
+    def hot_reload(self, hot_reload_time: float = 1) -> None:
```

Comment:
1. I think this (and also process_events and SyncedWorkSpaceHandler) shouldn't be a part of this class. I think we should create a separate python file for them.
2. Name is incorrect. It is not reloading anything.